stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt

```diff
@@ -10,11 +10,12 @@ import io.getstream.chat.android.ui.R
 import io.getstream.chat.android.ui.TransformStyle
 import io.getstream.chat.android.ui.common.extensions.internal.dpToPx
 import io.getstream.chat.android.ui.common.extensions.internal.getColorCompat
+import io.getstream.chat.android.ui.common.extensions.internal.getDimension
 import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
 import io.getstream.chat.android.ui.common.extensions.internal.use
 import io.getstream.chat.android.ui.common.style.TextStyle
 
-public data class FileAttachmentsViewStyle(
+public data class FileAttachmentViewStyle(
```

Comment:
Decided to do this without deprecation as these changes are recent and evident. In most cases (attrs, transformer, etc.) the singular form is used, therefore changing the name here

Reply:
Let's just mention this in CHANGELOG and we are good to go